mteb/evaluation/MTEB.py

```diff
@@ -58,6 +59,11 @@ def __init__(
 
         if tasks is not None:
             self._tasks = tasks
+            if (
+                not isinstance(tasks[0], AbsTask) and not isinstance(tasks[0], str)
+            ):  # So that passing in a list of Benchmarks works too. Trying to import Benchmark causes circular import.
```

Comment:
Hmm, I think we should use Benchmark here, why do we see the circular import? (something we should fix?)

Sidenote: https://github.com/embeddings-benchmark/mteb/issues/1272

Reply:
Done. And also commented on the tagged issue.